docs/admin/users/idp-sync.md

```diff
@@ -110,13 +107,10 @@ Below is an example that uses the `groups` claim and maps all groups prefixed by
 }
 ```
 
-<blockquote class="admonition note">
-
-You must specify Coder group IDs instead of group names. The fastest way to find
-the ID for a corresponding group is by visiting
-`https://coder.example.com/api/v2/groups`.
-
-</blockquote>
+> [!NOTE]
+> You must specify Coder group IDs instead of group names. The fastest way to find
+> the ID for a corresponding group is by visiting
+> `https://coder.example.com/api/v2/groups`.
```

Comment:
```suggestion
> [!IMPORTANT]
> You must specify Coder group IDs instead of group names. The fastest way to find
> the ID for a corresponding group is by visiting
> `https://coder.example.com/api/v2/groups`.
```

Looks important